File: app/src/main/java/com/lynn/simplerecyclerview/BaseApplication.java
package com.lynn.simplerecyclerview;

import android.app.Application;

import com.facebook.drawee.backends.pipeline.Fresco;

/**
 * Created by Lynn.
 */

public class BaseApplication extends Application {
    private static BaseApplication instance;

    @Override
    public void onCreate() {
        super.onCreate();
        this.instance = this;
        Fresco.initialize(this);
    }

    public static BaseApplication getInstance() {
        return instance;
    }
}

File: app/src/main/java/com/lynn/simplerecyclerview/BaseApplication.kt
package com.lynn.simplerecyclerview

import android.app.Application

import com.facebook.drawee.backends.pipeline.Fresco

/**
 * Created by Lynn.
 */

class BaseApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        instance = this
        Fresco.initialize(this)
    }

    companion object {
        var instance : BaseApplication? = null
            private set
    }
}
